components/vf-dropdown/vf-dropdown.njk

```diff
@@ -1,5 +1,10 @@
-{# Make sure any variables are listed inside the following if statement #}
-
+{%- if context %}
```

Comment:
@khawkins98 this njk templates is new to me. Can you explain why its necessary to add the declarations even if works without them?

Reply:
https://github.com/visual-framework/vf-core/blob/ad90662b87148957d29dda623fc3f5335bb0d96f/tools/vf-component-generator/templates/_component.njk#L1-L4

This approach is not required for all (or most) situations, but sometimes a `context` object is passed (or inherited) from a parent template, this ensure the necessary settings are available to the template.